t0/seqio_tasks/tasks.py

```diff
@@ -0,0 +1,418 @@
+    "d4_train",
+    [task for task in d4_train_mixture["BASE"] if task not in TASK_BLACKLIST],
+    default_rate=lambda t: mixture_cap[t.name],
+)
+
+# seqio.MixtureRegistry.add(
+#     "gpt_train",
+#     [task for task in gpt_train_mixture if task not in TASK_BLACKLIST],
+#     default_rate=lambda t: mixture_cap[t.name],
+# )
+#
+# seqio.MixtureRegistry.add(
+#     "sglue_train",
+#     [task for task in sglue_train_mixture if task not in TASK_BLACKLIST],
+#     default_rate=lambda t: mixture_cap[t.name],
+# )
+
+seqio.MixtureRegistry.add(
+    "d4_gpt_eval_train",
+    [task for task in d4_train_mixture["BASE"] + d4_train_mixture["GPT_EVAL"] if task not in TASK_BLACKLIST],
+    default_rate=lambda t: mixture_cap[t.name],
+)
+
+seqio.MixtureRegistry.add(
+    "d4_gpt_sglue_train",
+    [task for task in d4_train_mixture["BASE"] + d4_train_mixture["GPT_EVAL"] + d4_train_mixture["SGLUE"] if task not in TASK_BLACKLIST],
+    default_rate=lambda t: mixture_cap[t.name],
+)
+
+# seqio.MixtureRegistry.add(
+#     "d4_eval",
+#     [task for task in d4_eval_mixture if task not in TASK_BLACKLIST],
+#     default_rate=functools.partial(seqio.mixing_rate_num_examples, maximum=500_000),
+# )  # eval mixture does not need to be capped
+
+
+seqio.MixtureRegistry.add(
+    "d4_score_eval",
+    [
+        task
+        for task in seqio.TaskRegistry.names()
+        if task.endswith("_score_eval")
+        and task.split("_score_eval")[0] in d4_eval_mixture["BASE"]
+        and task.split("_score_eval")[0] not in TASK_BLACKLIST
+    ],
+    default_rate=functools.partial(seqio.mixing_rate_num_examples, maximum=500_000),
+)
+
+# # Train tasks we don't care about evaluating on
+# D4_TRAIN_SKIP_EVAL = [
+#     "paws_labeled_final",
+#     "adversarial_qa_dbidaf",
+#     "adversarial_qa_dbert",
+#     "duorc_ParaphraseRC",
+#     "dream",
+#     "amazon_polarity",
+#     "app_reviews",
+#     "imdb",
+#     "wiki_bio",
+#     "gigaword",
+#     "multi_news",
+#     "samsum",
+#     "dbpedia_14",
+#     "trec",
+# ]
+
+# seqio.MixtureRegistry.add(
+#     "d4_train_eval",
+#     [
+#         task
+#         for task in d4_train_mixture
+#         if task not in TASK_BLACKLIST
+#         and not any([skip in task for skip in D4_TRAIN_SKIP_EVAL])
+#         and task in all_original_tasks
+#     ],
+#     default_rate=lambda t: mixture_cap[t.name],
+# )
+#
+# seqio.MixtureRegistry.add(
+#     "d4_train_score_eval",
+#     [
+#         task
+#         for task in seqio.TaskRegistry.names()
+#         if task.endswith("_score_eval")
+#         and task.split("_score_eval")[0] in d4_train_mixture
+#         and task.split("_score_eval")[0] not in TASK_BLACKLIST
+#         and task not in D4_TRAIN_SCORE_EVAL_TASK_BLACKLIST
+#         and not any([skip in task for skip in D4_TRAIN_SKIP_EVAL])
+#         and task.split("_score_eval")[0] in all_original_tasks
+#     ],
+#     default_rate=functools.partial(seqio.mixing_rate_num_examples, maximum=500_000),
+# )
```

Comment:
I correct what i initially discussed with you: let's keep this mixture, this is the one we used for checkpoint selection in the t zero paper